src/DssSpell.sol

```diff
@@ -106,226 +99,292 @@ contract DssSpellAction is DssAction {
+        DssExecLib.setIlkStabilityFee("WSTETH-A", THREE_PT_SEVEN_FOUR_PCT_RATE, /* doDrip = */ true);
+
+        // Set WSTETH-B Stability Fee to 3.49%
+        DssExecLib.setIlkStabilityFee("WSTETH-B", THREE_PT_FOUR_NINE_PCT_RATE, /* doDrip = */ true);
+
+        // --- Spark Protocol Parameter Changes ---
+        // D3M Parameter Adjustments Poll: https://vote.makerdao.com/polling/QmWatYqy#poll-detail
+        // Executive Proxy Poll: https://vote.makerdao.com/polling/Qmc9fd3j#poll-detail
+        // Onboard rETH Poll: https://vote.makerdao.com/polling/QmeEV7ph#vote-breakdown (Inside Proxy Spell)
+        // DAI Interest Rate Strategy Poll: https://vote.makerdao.com/polling/QmWodV1J#poll-detail (Inside Proxy Spell)
+        // Forum: https://forum.makerdao.com/t/2023-05-24-spark-protocol-updates/20958
+        DssExecLib.setIlkAutoLineParameters("DIRECT-SPARK-DAI", /* line */ 20 * MILLION, /* gap */ 20 * MILLION, /* ttl */ 8 hours);
+        DssExecLib.authorize(SPARK_PROXY, DssExecLib.esm());
+        ACLManagerLike(SPARK_ACL_MANAGER).addPoolAdmin(SPARK_PROXY);
+        ProxyLike(SPARK_PROXY).exec(SPARK_SPELL, abi.encodeWithSignature("execute()"));
+        DssExecLib.setChangelogAddress("EXEC_PROXY_SPARK", SPARK_PROXY);
+
+        // --- Non-Scope Defined Parameter Adjustments ---
+        // Poll: https://vote.makerdao.com/polling/QmQXhS3Z#poll-detail
+        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-2-non-scope-defined-parameter-changes-may-2023/20981
+
+        // Increase rETH-A line to 50 million DAI
+        // Increase rETH-A gap to 5 million DAI
+        DssExecLib.setIlkAutoLineParameters("RETH-A", /* line */ 50 * MILLION, /* gap */ 5 * MILLION, /* ttl */ 8 hours);
+
+        // Increase rETH-A Stability Fee to 3.74%
+        DssExecLib.setIlkStabilityFee("RETH-A", THREE_PT_SEVEN_FOUR_PCT_RATE, true);
+
+        // Increase CRVV1ETHSTETH-A Stability Fee to 4.24%
+        // NOTE: disabled for goerli because the collateral is not on the chain
+        // DssExecLib.setIlkStabilityFee("CRVV1ETHSTETH-A", FOUR_PT_TWO_FOUR_PCT_RATE, true);
+
+        // Increase WBTC-A Stability Fee to 5.80%
+        DssExecLib.setIlkStabilityFee("WBTC-A", FIVE_PT_EIGHT_PCT_RATE, true);
+
+        // Increase WBTC-B Stability Fee to 6.30%
+        DssExecLib.setIlkStabilityFee("WBTC-B", SIX_PT_THREE_PCT_RATE, true);
+
+        // Increase WBTC-C Stability Fee to 5.55%
+        DssExecLib.setIlkStabilityFee("WBTC-C", FIVE_PT_FIVE_FIVE_PCT_RATE, true);
+
+        // --- RWA015 (BlockTower Andromeda) ---
+        // Poll: https://vote.makerdao.com/polling/QmbudkVR#poll-detail
+        // Forum links:
+        //   - https://forum.makerdao.com/t/mip90-liquid-aaa-structured-credit-money-market-fund/18428
+        //   - https://forum.makerdao.com/t/project-andromeda-risk-legal-assessment/20969
+        //   - https://forum.makerdao.com/t/rwa015-project-andromeda-technical-assessment/20974
+        onboardRWA015A();
```

Comment:
Why was the bootstrapping inlined and the onboarding was not?
I'd say we either inline both or none of them.